public/app/plugins/datasource/loki/querybuilder/operations.ts

```diff
@@ -410,181 +420,3 @@ export function getOperationDefinitions(): QueryBuilderOperationDef[] {
 
   return list;
 }
-
```

Comment:
I cleaned this file up and moved all of these function to operationUtils.ts, and kept here only operations.